packages/api/src/router/widgets/indexer-manager.ts

```diff
@@ -0,0 +1,67 @@
+import { TRPCError } from "@trpc/server";
+
+import { ProwlarrIntegration } from "@homarr/integrations";
+import { logger } from "@homarr/log";
+
+import { createManyIntegrationMiddleware } from "../../middlewares/integration";
+import { createTRPCRouter, publicProcedure } from "../../trpc";
+
+export const indexerManagerRouter = createTRPCRouter({
+  indexers: publicProcedure
```

Comment:
This also get's the status, correct? If yes, I strongly suggest you to rename to getIndexerStatus or getIndexers